libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -277,4 +299,25 @@ public void onServiceDisconnected(ComponentName className) {
       serviceBound = false;
     }
   };
+
+  private boolean isUserAgentValid(String userAgent) {
+    if (userAgent != null && !TextUtils.isEmpty(userAgent)) {
```

Comment:
If we’re going to use `TextUtils.isEmpty` the `null` check is not needed, did you :eyes: what `TextUtils.isEmpty` does internally?
If you check the implementation the only thing is doing is `blablah != null && !blablah.isEmpty()`, so I’d rather remove `TextUtils` (Android class that could affect Unit tests) and manage it by ourselves.